src/misc/windowing.jl

```diff
@@ -74,6 +72,38 @@ function windowmap!(f::Function, out::AbstractVector, x::AbstractVector; kwargs.
     return out
 end
 
+"""
+    slidebracket(t::AbstractVector, bracketing::Symbol; kwargs...)
+
+Return bracketing of `t` based on [`windowmap`](@ref).
+The type of bracketing used can be chosen among:
+- `:left`: the last element of the window is returned.
+- `:center`: the center element of the window is returned.
+- `:right`: the first element of the window is returned.
+
+## Keyword arguments:
+- `width::Int` of the sliding window.
+- `stride::Int` of the sliding window.
+"""
+function slidebracket(t::AbstractVector, bracketing::Symbol; kwargs...)
+    if bracketing == :left
+        @info "Left-sided bracketing chosen: best suited for "*
```

Comment:
this isn't where this information should be displayed. In the documentation of the function instead. Don't use `@info`.